### DISCUSSION TITLE
404 page not found error

### DISCUSSION THREAD

**Discussion by Benji4927MC** [community/community#110756] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

hey my github site isn't working ([benji4927mc.github.io](benji4927mc.github.io)) and im wondering how to fix it?
plz help im new here btw just copy the link
![Screenshot 2024-03-02 2 53 05 PM](https://github.com/community/community/assets/150076776/3f036fb3-e52d-44a8-bbf9-13f859c5f1dd)

**Comment by Benji4927MC** [comment]
btw just copy the link i pressed it and it just takes you to: https://github.com/orgs/community/discussions/benji4927mc.github.io

**Comment by jfullstackdev** [comment]
your `index.html` is inside another folder, which is not even being used in deployment, try to put that at the root of your repo, make sure you turn on GitHub Pages

![image](https://github.com/community/community/assets/47092464/92a46aee-9ed0-42ae-a859-7ba9bb6c1eea)
